paper/appendix.tex

```diff
@@ -0,0 +1,1042 @@
+{\huge \textbf{Appendix A: User Study}}\\\\\\\\  % Minye
+{\large \textbf{A.1: Interview Tasks }} \\\\
+\section{Interview Tasks}\label{appendix:tasks}
+This part is about interview tasks we prepared to conduct our user study.
```

Comment:
Please also maybe note that the tasks were modified and also that the third task was only added after the first user study